scio-avro/src/main/scala/com/spotify/scio/avro/types/TypeProvider.scala

```diff
@@ -380,18 +382,15 @@ private[types] object TypeProvider {
    * This is used to mitigate lack of support for Scala macros in IntelliJ.
    */
   private def shouldDumpClassesForPlugin =
-    sys.props("bigquery.plugin.disable.dump") == null ||
-      !sys.props("bigquery.plugin.disable.dump").toBoolean
+    !AvroSysProps.DisableDump.value("true").toBoolean
 
   private def getBQClassCacheDir: Path = {
     // TODO: add this as key/value settings with default etc
-    if (sys.props("bigquery.class.cache.directory") != null) {
-      Paths.get(sys.props("bigquery.class.cache.directory"))
-    } else {
+    AvroSysProps.Debug.valueOption[String].map(Paths.get(_)).getOrElse {
       Paths
-        .get(sys.props("java.io.tmpdir"))
-        .resolve(sys.props("user.name"))
-        .resolve("bigquery-classes")
```

Comment:
@nevillelyh any reason why this was using the same folder? I guess we can put them on a different one right?
/cc @jto @ClaireMcGinty

Reply:
Just laziness, so that `scio-idea-plugin` works with both Avro & BQ macro out of the box (it only looks in `bigquery-classes`).

Reply:
@nevillelyh should we revert the name back for now?

Reply:
I would rather revert to the old name just to be sure we don't break anyone's intellij. We'll have enough support to do without blowing up IDEs...